elements/auras.lua

```diff
@@ -441,14 +412,15 @@ local function UpdateAuras(self, event, unit, updateInfo)
 				--]]
 				table.sort(auras.sortedBuffs, auras.SortBuffs or auras.SortAuras or SortAuras)
 
-				for i = 1, #auras.sortedBuffs do
+				count = math.min(count, #auras.sortedBuffs)
+				for i = 1, count do
 					updateAura(auras, unit, auras.sortedBuffs[i], i)
 				end
 			end
 
-			local offset = #auras.sortedBuffs
+			local offset = count
 
-			if(auras.gap and #auras.sortedDebuffs > 0) then
+			if(auras.gap and count > 0) then
```

Comment:
The gap is now always shown though, if buffs are also shown. Since it's an invisible button I guess it does not hurt, otherwise I'd have to move the Show/Hide() call to after updating the debuffs.